.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "themes/drone-hugo-theme"]
```

Comment:
Please don't add it as a submodule, we want to keep always the latest theme version.

Reply:
I would personally prefer a simple git clone before the Hugo plugin step.